src/components/OperationTag.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Original file: https://github.com/Kong/swagger-ui/blob/main/src/core/components/operation-tag.jsx
+ * @prettier
+ */
+
+import React from "react"
+import { createDeepLinkPath, escapeDeepLinkPath, sanitizeUrl } from '../helpers/helpers'
+
+export default class OperationTag extends React.Component {
+  handleKeypress = (event, isShownKey, showTag) => {
+    const { layoutActions } = this.props
+
+    if (event.nativeEvent.code === "Enter" || event.nativeEvent.code === "Space") {
+      layoutActions.show(isShownKey, showTag)
+    }
+  }
+
+  render() {
+    const {
+      tagObj,
+      tag,
+      children,
+
+      layoutSelectors,
+      layoutActions,
+      getConfigs,
+      getComponent,
+    } = this.props
+
+    let { docExpansion } = getConfigs()
+
+    const Collapse = getComponent("Collapse")
+    const Markdown = getComponent("Markdown")
+    const DeepLink = getComponent("DeepLink")
+    const Link = getComponent("Link")
+
+    let tagDescription = tagObj.getIn(["tagDetails", "description"], null)
+    let tagExternalDocsDescription = tagObj.getIn(["tagDetails", "externalDocs", "description"])
+    let tagExternalDocsUrl = tagObj.getIn(["tagDetails", "externalDocs", "url"])
+
+    let isShownKey = ["operations-tag", tag]
+    let showTag = layoutSelectors.isShown(isShownKey, docExpansion === "full" || docExpansion === "list")
+
+    return (
+      <div
```

Comment:
Any reason why we changed from `section` to `div`?

Reply:
Accident - let me change it back.